server/src/main/java/org/apache/iotdb/db/query/dataset/ShowTimeSeriesResult.java

```diff
@@ -134,13 +151,25 @@ public static ShowTimeSeriesResult deserialize(ByteBuffer buffer) {
     result.encoding = ReadWriteIOUtils.readString(buffer);
     result.compressor = ReadWriteIOUtils.readString(buffer);
```

Comment:
we have the enums of these strings, I suggest that we use the enums to serialize, u can find it in :
```
package org.apache.iotdb.tsfile.file.metadata.enums;
```